styles/widgets/generic/scheduler.generic.less

```diff
@@ -659,3 +659,7 @@
 .dx-scheduler-agenda-nodata {
     font-size: @GENERIC_SCHEDULER_WORKSPACE_INFO_TEXT_SIZE;
 }
+
+.dx-scheduler-appointment-tooltip-wrapper .dx-overlay-content .dx-popup-content .dx-list-item {
```

Comment:
By the way it is generic

Reply:
There was a feedback from Dmitriy Kharkov for generic theme tooltip in scheduler